src/App.tsx

```diff
@@ -17,59 +17,112 @@ const goodsFromServer = [
 
 enum SortType {
   NONE,
-  ALPABET,
+  ALPHABET,
   LENGTH,
 }
 
```

Comment:
you deleted function getReorderGoods...
As far as I get it we need to use it for main functionality

Reply:
Agree with Yana here